src/fhir/document.ts

```diff
@@ -1,28 +1,44 @@
+import { useEffect, useState } from "react";
 import { useIncludeBasics } from "./basic";
-import { searchCommonRecords } from "./search-helpers";
-import { PatientModel, useFeatureFlaggedQueryWithPatient } from "..";
+import { DocumentModel } from "./models/document";
+import { PatientModel, useQueryWithPatient } from "..";
 import { applyDocumentFilters } from "@/components/content/document/helpers/filters";
 import { CTWRequestContext } from "@/components/core/providers/ctw-context";
-import { useFQSFeatureToggle } from "@/hooks/use-feature-toggle";
 import { createGraphqlClient, fqsRequest } from "@/services/fqs/client";
 import { DocumentReferenceGraphqlResponse, documentsQuery } from "@/services/fqs/queries/documents";
 import { orderBy } from "@/utils/nodash";
 import { QUERY_KEY_PATIENT_DOCUMENTS } from "@/utils/query-keys";
-import { Telemetry } from "@/utils/telemetry";
+import { Telemetry, withTimerMetric } from "@/utils/telemetry";
 
-export function usePatientDocuments() {
-  const fqs = useFQSFeatureToggle("documents");
+export function usePatientTopLevelDocuments() {
+  const { data, isError, isFetching, isLoading } = usePatientDocuments();
+  const [filteredData, setFilteredData] = useState([] as DocumentModel[]);
+
+  useEffect(() => {
+    const filteredDocuments = orderBy(
+      applyDocumentFilters(data),
+      [(document) => document.resource.content[0].attachment.creation || ""],
+      ["desc"]
+    );
```

Comment:
Another nit, but a cool nit this time
```suggestion
    const filteredDocuments = orderBy(
      applyDocumentFilters(data),
      "resource.content[0].attachment.creation",
      ["desc"]
    );
```